packages/cli/src/reporters/abstract-list.ts

```diff
@@ -94,16 +108,20 @@ export default abstract class AbstractListReporter implements Reporter {
         status.push(sourceFile ? indentString(titleString, 2) : titleString)
       }
     }
+
     if (!opts.skipCheckCount) {
       status.push('')
       status.push([
+        counts.scheduling ? chalk.bold.blue(`${counts.scheduling} scheduling`) : undefined,
```

Comment:
Super nit: I think that it could be nice to have the `scheduling` count right near the `pending` count. Scheduling and pending are kind of similar, so I think it's nice to have them side by side.